<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_nomeRestaurante">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#000000;&quot;&gt;Restaurante Fogo de Riba&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sobre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SOBRE NÓS</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Imagens/aboutIcon.jfif</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cardapio">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>  ACESSE NOSSO CARDÁPIO</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Imagens/cardapio2Icon.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fechar">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Imagens/leaveIcon.jfif</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pedidos">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  FAÇA JÁ O SEU PEDIDO!</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Imagens/cardapioIcon.jfif</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_fundo1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>591</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Imagens/gaviao.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fundo2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fraseWow">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#ffffff;&quot;&gt;Fogo de riba no fogão,&lt;br/&gt;calor no coração!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label_nomeRestaurante</zorder>
   <zorder>label_fundo1</zorder>
   <zorder>label_fundo2</zorder>
   <zorder>label_fraseWow</zorder>
   <zorder>pushButton_cardapio</zorder>
   <zorder>pushButton_pedidos</zorder>
   <zorder>pushButton_sobre</zorder>
   <zorder>pushButton_fechar</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Imagens/Imagens.qrc"/>
 </resources>
 <connections/>
</ui>
